### PR TITLE
Version up to V13

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

V13 updates 

## Changes

This PR updates v13.0.x to v13. 

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [x] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,18 +6,20 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+  release).  The success of this operation is dependent on system ingest load.
 
   In rare cases where data-persistence deployment fails because the
   `postgres-db-migration` times out, it may be required to manually remove the
   index and then redeploy:
 
   ```text
-  DROP INDEX IF EXISTS postgres-db-migration;
+  > DROP INDEX IF EXISTS postgres-db-migration;
   DROP INDEX
```
Comment (npauzenga) [inline CHANGELOG.md:23]: Is `DROP INDEX` the output of the command? Should we remove that for clarity?

**Review by npauzenga — APPROVED** [review]
LGTM! Left the most minor comment I could muster on the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,18 +6,20 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [v12.0.0] 2022-06-13
```
Comment (botanical) [inline CHANGELOG.md:9]: Should this be v13? 
```suggestion
## [v13.0.0] 2022-06-13
```

File: CHANGELOG.md (hunk 1 above)
Comment (Jkovarik) [reply CHANGELOG.md]: Excellent point

928e8c3b1

File: CHANGELOG.md (hunk 2 above)
Comment (Jkovarik) [reply CHANGELOG.md]: 😓  Indeed!  928e8c3b1

**Pull request merged**